src/Codeception/Module/Symfony2.php

```diff
@@ -382,7 +391,7 @@ public function seeInCurrentRoute($routeName)
         }
 
         try {
-            $matchedRouteName = $router->match($this->grabFromCurrentUrl())['_route'];
+            $matchedRouteName = $router->match(explode('?', $this->grabFromCurrentUrl())[0])['_route'];
         } catch (\Exception\ResourceNotFoundException $e) {
```

Comment:
The correct exception should be the same as Line 364: `\Symfony\Component\Routing\Exception\ResourceNotFoundException`